### ISSUE TITLE
BUG: working with iframe canvas is very buggy and unstable

### ISSUE THREAD

**Issue by myownstandard** [zsviczian/obsidian-excalidraw-plugin#1227]
**Your environment**
SYSTEM INFO:
	Obsidian version: v1.3.7
	Installer version: v1.1.9
	Operating system: Windows 10 Enterprise LTSC 2021 10.0.19044
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 53
	Plugins enabled: 42
		1: Dataview v0.5.56
		2: Excalidraw v1.9.8
		3: Kanban v1.5.3
		4: Calendar v1.5.10
		5: Templater v1.16.0
		6: Outliner v4.7.1
		7: Natural Language Dates v0.6.1
		8: Admonition v9.3.3
		9: Style Settings v1.0.6
		10: Tag Wrangler v0.5.11
		11: Emoji Toolbar v0.4.1
		12: Recent Files v1.3.6
		13: DB Folder v3.4.0
		14: Toggle Meta Yaml v1.2.0
		15: Auto Link Title v1.4.1
		16: Periodic Notes v0.0.17
		17: Audio Player v0.1.1
		18: Status Bar Pomodoro Timer v0.1.14
		19: Projects v1.16.3
		20: Metadata Menu v0.4.21
		21: TagFolder v0.16.9
		22: Hover Editor v0.11.13
		23: Link Exploder v0.3.0
		24: Quiet Outline v0.3.15
		25: Omnisearch v1.14.2
		26: DataLoom v8.0.0
		27: Floating Search v3.3.1
		28: Custom Frames v2.4.6
		29: Tasks v4.3.0
		30: Full Calendar v0.10.7
		31: Canvas Filter v0.9.2
		32: Commander v0.5.0
		33: Persistent Links v0.2.0
		34: Image Toolkit v1.3.1
		35: Obsidian42 - BRAT v0.7.0
		36: Canvas View v1.1.1
		37: Emoji Magic v0.2.2
		38: Completr v3.2.0
		39: Obsidian42 - Strange New Worlds (SNW) v1.2.2
		40: Contextual Typography v2.2.5
		41: Charts View v1.2.5
		42: obsidian markmind v1.7.5

**Describe the bug**
Bugs while using iframe with Canvas (all info in video)

**To Reproduce**
1. Create New Excalidraw drawing
2. Insert Canvas
3. Try to interacte with it

**Expected behavior**
Canvas actions should work well : type text, drag objects on canvas , mouse coursor selection under the coursor

**Screenshots**
https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/78643199/6800cfb4-6097-49c7-94eb-7f3c3a662d75

**Additional context**
Windows 10
Obsidian 1.3.7
Excalidraw plugin 1.9.8

**Comment by zsviczian** [comment]
Yes, I am aware, however, at this time I don't treat this as a priority.
Based on my testing canvas works OK at Excalidraw 100% zoom rate. The issue has to do with the translation of pointer events to canvas. I don't yet know what the solution might be.
